internal/pkg/service/cli/cmd/remote/file/download.go

```diff
@@ -17,7 +17,6 @@ func DownloadCommand(p dependencies.Provider) *cobra.Command {
 		Use:   `download [file]`,
 		Short: helpmsg.Read(`remote/file/download/short`),
 		Long:  helpmsg.Read(`remote/file/download/long`),
-		Args:  cobra.MinimumNArgs(1),
```

Comment:
Myslím, že by tady mělo být `cobra.MaximumNArgs(1)`

Reply:
Aha, ani nevím že tam taková možnost je (nejspíš jsme ji ani nepoužívali), ale určitě tím nic tím nezkazím. 🙂 Díky.

Reply:
Zkus 
```
remote table import --incremental-load true --without-headers true --storage-api-host %%TEST_KBC_STORAGE_API_HOST%% --storage-api-token %%TEST_KBC_STORAGE_API_TOKEN%% in.c-test.accounts %%TEST_FILE_TEST1_ID%% 
```
(dát ty flags před `[table] [file]`)

Reply:
Taky ještě mě napadá vyhodit ty `true`, protože pro boolean flagy jen presence stačí aby byl true, nevím, jestli to vůbec podporuje true/false jako hodnoty